pkgs/by-name/sw/swhkd/package.nix

```
@@ -0,0 +1,36 @@
+{lib, rustPlatform, fetchFromGitHub, pkg-config, pkgs }:
```

```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
, pkg-config
, systemd
, libgcc
}:
```

you should not need to import `pkgs`, arguments will be filled by callPackage.